packages/experiments-realm/components/grid.gts

```diff
@@ -44,22 +44,27 @@ export class CardsGrid extends GlimmerComponent<CardsGridSignature> {
           </:loading>
           <:response as |cards|>
             {{#each cards key='url' as |card|}}
-              <li
-                class='{{@selectedView}}-view-container'
-                {{@context.cardComponentModifier
-                  cardId=card.url
-                  format='data'
-                  fieldType=undefined
-                  fieldName=undefined
-                }}
-              >
-                <CardContainer
-                  class='card'
-                  @displayBoundaries={{not (eq @selectedView 'card')}}
+              <li class='{{@selectedView}}-view-container'>
+                <div
+                  {{@context.cardComponentModifier
+                    cardId=card.url
+                    format='data'
+                    fieldType=undefined
+                    fieldName=undefined
+                  }}
```

Comment:
You can probably just put this modifier on the CardContainer below and not need the extra div. Otherwise the changes you made look good.

Reply:
Yea thanks. I was worried of previous issues with the overlay but I think having an extra. div will cause more confusion. I dont see any issues atm